maliput/tests/segment_test.rs

```diff
@@ -35,7 +35,7 @@ fn segment_api() {
     let road_geometry = road_network.road_geometry();
 
     let segment_id = String::from("0_0");
-    let segment = road_geometry.get_segment(&segment_id);
+    let segment = road_geometry.get_segment(&segment_id).unwrap();
```

Comment:
Can we add a test for when segment id is not found?

Reply:
Done.